README.md

```diff
@@ -37,6 +37,9 @@ Eclipse Theia Blueprint is ***not*** **a production-ready product**. Therefore,
 
 ## Development
 
+### Requirements
+Please check Theia's [prerequisites](https://github.com/eclipse-theia/theia/blob/master/doc/Developing.md#prerequisites), and keep the node version alligned between the blueprint and that of the referenced theia version.
```

Comment:
nitpick: `alligned` -> `aligned`

Reply:
while you're there, I would remove "the" in "between the blueprint" and capitalise Blueprint. and Theia. 
i.e.: 
"and keep the node version allgned between Blueprint and that of the referenced Theia version."

Reply:
fixed the typos, also made title case consistent for headings